ui/src/app/header/header.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { CartService } from 'src/Shared/services/cart.service';
+import { LoginServiceService } from 'src/Shared/services/login-service.service';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.sass']
+})
+export class HeaderComponent implements OnInit {
+  isModal: boolean = false;
+  noOfItems: number = 0;
+  setLogOut: boolean = false;
```

Comment:
How about `isLoggedOut` or `isLoggedIn`?

Reply:
removing the extra one now